timesync/clock.go

```diff
@@ -137,10 +137,13 @@ func (t *NodeClock) tick() {
 		d := t.lastTicked.Difference(layer)
 		tickDistance.Observe(float64(-d))
 	case layer.Difference(t.lastTicked) > 1:
-		t.log.With().Warning("clock skipped layers",
-			log.Stringer("layer", layer),
-			log.Stringer("last_ticked_layer", t.lastTicked),
-		)
+		// don't warn right after fresh startup
+		if t.lastTicked != 0 {
+			t.log.With().Warning("clock skipped layers",
+				log.Stringer("layer", layer),
+				log.Stringer("last_ticked_layer", t.lastTicked),
+			)
+		}
```

Comment:
You can combine the check into the case:

```suggestion
case layer.Difference(t.lastTicked) > 1 && t.lastTicked > 0:
		t.log.With().Warning("clock skipped layers",
			log.Stringer("layer", layer),
			log.Stringer("last_ticked_layer", t.lastTicked),
		)
```

Reply:
@fasmat that changes the behavior of the code. read the following context.

Reply:
I'm just suggesting that instead of

```go
if layer.Difference(t.lastTicket) > 1 {
    if t.lastTicket != 0 {
        t.og.With.Warning("...")
    }
}
```

to write:

```go
if layer.Difference(t.lastTicket) > 1 && if t.lastTicket != 0 {
    t.log.With().Warning("...")
}
```

which should behave exactly the same?

Reply:
Regarding the code that comes afterwards; I think it should also not be executed if `t.lastTick` is 0 if that's where the confusion comes from 🙂, sorry for not writing a complete code example.

Reply:
why wouldn't you want to execute `tickDistance.Observe()`? I'm trying to change a log here, not any other behavior.

Reply:
i see, that's only used in metrics... didn't check that